docs/examples/InputTypes.js

```diff
@@ -16,7 +16,9 @@ const inputTypeInstance = (
     </Input>
     <Input type='textarea' label='Text Area' placeholder='textarea' />
     <Input type='static' value='Static Text' />
-    <Input type='submit' value='Submit button' />
+    <ButtonInput value='Button Input' />
+    <ButtonInput type='reset' value='Reset Button' />
+    <ButtonInput type='submit' value='Submit Button' />
```

Comment:
Is this intentional when the `ButtonInput` has children?

![screen shot 2015-05-13 at 11 12 05 am](https://cloud.githubusercontent.com/assets/736870/7616356/f8ba86a8-f960-11e4-9935-96024edbc6bd.png)